sovtoken/sovtoken/test/helpers/helper_node.py

```diff
@@ -29,16 +30,21 @@ def get_last_ledger_transaction_on_nodes(self, ledger_id):
 
         return transactions
 
-    def get_token_req_handler(self):
-        """ Get the token request handler of the first node. """
-        return self._nodes[0].get_req_handler(ledger_id=TOKEN_LEDGER_ID)
+    @property
+    def xfer_handler(self):
+        """ Get the xfer request handler of the first node. """
+        return self._nodes[0].write_manager.request_handlers[XFER_PUBLIC][0]
 
-    def get_domain_req_handler(self):
-        """ Get the domain request handler of the first node. """
-        return self._nodes[0].get_req_handler(ledger_id=DOMAIN_LEDGER_ID)
+    @property
+    def mint_handler(self):
+        """ Get the xfer request handler of the first node. """
+        return self._nodes[0].write_manager.request_handlers[MINT_PUBLIC][0]
```

Comment:
Please, use lookup instead of indexing ("isinstance" call )

Reply:
fixed